autoload/lsp/diag.vim

```diff
@@ -41,6 +46,15 @@ def DiagsRefreshSigns(lspserver: dict<any>, bnr: number)
     signs->add({id: 0, buffer: bnr, group: 'LSPDiag',
 				lnum: lnum,
 				name: DiagSevToSignName(diag.severity)})
+
+    if has('patch-9.0.1157') && opt.lspOptions.showDiagWithVirtualText
+        prop_add( lnum, 0, {
+            'bufnr': bnr,
```

Comment:
Can you remove the single quotes from the key names?  It is not needed in Vim9 script.

Reply:
> > I don't understand why though, as these shouldn't affect each other?
> 
> Did you try with the latest plugin? A redraw call was added before displaying the popup.

I tried it out on d161db7, see https://asciinema.org/a/0VAnghdBa73OatXW2lCC7vnDV

Reply:
fix added in https://github.com/yegappan/lsp/pull/187/commits/6a7464d2dfa68fa5e4525e876712bcbed5f0300b

Reply:
> > > I don't understand why though, as these shouldn't affect each other?
> > 
> > 
> > Did you try with the latest plugin? A redraw call was added before displaying the popup.
> 
> I tried it out on [d161db7](https://github.com/yegappan/lsp/commit/d161db7c5d8ed8e2a4d73fa31e67dc54221f4eaa), same problem, see https://asciinema.org/a/0VAnghdBa73OatXW2lCC7vnDV

I think there is an error in the virtual text function at the editor level.
 I noticed that the virtual text doesn't always work correctly with pop-up dialogs.